nightfall-deployer/contracts/Challenges.sol

```diff
@@ -34,8 +34,7 @@ contract Challenges is Stateful, Key_Registry, Config {
         Block memory priorBlockL2, // the block immediately prior to this one
         Transaction[] memory priorBlockTransactions, // the transactions in the prior block
         Block memory blockL2,
-        Transaction[] memory transactions,
-        bytes32 salt
```

Comment:
why are you removing salt from the challenge? We would need it as part of `checkCommit(msg.data)`